docs/rebuild_admin.rst

```diff
@@ -0,0 +1,407 @@
+
+  Host mon
+    User <admin-username>
+    Hostname <monitor-server-ip>
+    ProxyCommand none
+
+
+Finally, test direct SSH access from the terminal, using the commands ``ssh app`` and 
+``ssh mon``. It should be possible to connect without entering a password.
+
+Step 4: Retrieve SecureDrop configuration info from the servers
+===============================================================
+
+In addition to the account and networking information retrieved from the servers
+so far, you'll need to retrieve the following files and info:
+
+ - Tor hidden service URLs and tokens
+ - GPG *Submission Public Key* and OSSEC public key
+ - OSSEC alert configuration details
+ - (Optional) HTTPS configuration details
+
+Retrieve Onion hidden service info
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to connect to the Tor hidden services used by your instance, you will
+need to copy their details to the *Admin Workstation*. To do so, run the following
+commands from a Terminal window:
+
+.. code:: sh
+
+ cd ~/Persistent/securedrop/install_files/ansible-base
+ echo "HidServAuth $(ssh app sudo cat /var/lib/tor/services/journalist/hostname)" \
+    > app-journalist-aths
+ echo "$(ssh app sudo cat /var/lib/tor/services/journalist/hostname)" > app-source-ths
+
+If your instance was set up to use SSH over TOR, you'll also need to copy over the details 
+of the SSH proxy hidden services, by running the following commands:
+
+.. code:: sh
+
+ echo "HidServAuth $(ssh app sudo cat /var/lib/tor/services/ssh/hostname)" > app-ssh-aths
+ echo "HidServAuth $(ssh mon sudo cat /var/lib/tor/services/mon/hostname)" > mon-ssh-aths
+
+Retrieve GPG Public Keys 
+~~~~~~~~~~~~~~~~~~~~~~~~
+There are 2 GPG public keys required by the instance configuration, that you will need to 
+copy to the new *Admin Workstation*.
+
+To copy the *Submission Public Key*, download it from the *Source Interface* and import
+it locally using the following commands:
+
+.. code:: sh
+
+ cd ~/Persistent/securedrop/install_files/ansible-base
+ curl http://$(cat app-source-ths)/journalist-key > SecureDrop.asc
+ gpg --import SecureDrop.asc
+
+Validate that the imported key's fingerprint matches the one on your 
+SecureDrop install. You can do this by first running the command:
+
+.. code:: sh
+
+ gpg -k --fingerprint 
+
+Then, compare the value with the fingerprint advertised by your *Source Interface*,
+using the command:
+
+.. code:: sh
+
+ curl http://$(cat app-source-ths)/metadata
+
+To copy the OSSEC alert public key, first list available keys on the monitor server:
+
+.. code:: sh
+ 
+ ssh mon sudo gpg --homedir=/var/ossec/.gnupg  -k
+
+Look for the key corresponding to the destination email address for OSSEC alert. Then, import it locally using the following commands (substituting the appropriate email address for ``alerts@example.com``:
+
+.. code:: sh
+
+ ssh mon sudo gpg --homedir=/var/ossec/.gnupg --export --armor alerts@example.com > ossec.asc
+ gpg --import ossec.asc
```

Comment:
Note that [the default filename is ossec.pub](https://github.com/freedomofpress/securedrop/blob/develop/admin/securedrop_admin/__init__.py#L331), so it may make sense to use that here as well to avoid confusion later.

Reply:
fixed